### ISSUE TITLE
Fix the makefile

### ISSUE THREAD

**Issue by robertsawko** [BenMql/coral#6]
Firstly, I was hit by a missing trailing slash for the directory. You may want to add `/` after `$(CORAL_ROOT)` in makefile. There may be a more robust way of doing this to avoid double slashes (`pathJoin` in Lmod), but I would assume it's better to have a double slash than a missing one as one results in an error and the other one doesn't.

I am stuck though on the -module option.

```
mpif90 -cpp -O2 -march=native -c /opt/intel/mkl/include/lapack.f90 -o /home/rsa/documents/reviews/to_review/miquel2021//coral/src/cheby_tools/lapack_module.o -M/home/rsa/documents/reviews/to_review/miquel2021//coral/.mod/
gfortran: error: unrecognized command-line option ‘-M/home/rsa/documents/reviews/to_review/miquel2021//coral/.mod/’
make: *** [makefile:28: /home/rsa/documents/reviews/to_review/miquel2021//coral/src/cheby_tools/lapack_module.o] Error 1
```

**Comment by BenMql** [comment]
Robert, 

Good suggestion, I added some `/` in the makefile.

Concerning the module option, the gnu compiler flag is `-J`. I have modified the instructions for setting local variables [here](https://github.com/BenMql/coral/wiki/1_Installation#defining-compilation-options-and-variables) so that to nudge the user towards using `/etc/set_env_examples/set_env_template.sh` as a starting point. This file contains more guidance than other scripts. Does the module path definition (around line 50) make sense?
